### ISSUE TITLE
Problems with Docker images >=4.0.2

### ISSUE THREAD

**Issue by hkorte** [holgerbrandl/kscript#361]
The new Docker images don't work for me. It sounds like problems with the alpine image:
```
$ docker run -i holgerbrandl/kscript:4.0.0 'println("Hello, world!")'
Hello, world!
$ docker run -i holgerbrandl/kscript:4.0.2 'println("Hello, world!")'
[kscript] [ERROR] Unsupported OS: linux-musl
$ docker run -i holgerbrandl/kscript:4.0.3 'println("Hello, world!")'
[kscript] [ERROR] Unsupported OS: linux-musl
```

**Comment by aartiPl** [comment]
Unfortunately getting right OSTYPE proves to be complex with so many different types reported by OS'es... :-( 
Nevertheless, I will try to create a fix and publish it soon.

@holgerbrandl - I don't want to trouble you with every small release. On the other hand, I don't know how to easily allow releases by different contributors in case you need to log in to other services to do that. I have shared my email in my Github account. Please drop me an email with your thoughts and share recipes on how to get access to e.g. sdkman, brew, docker, etc. Anyway, I think we can work out some way to do it correctly.

**Comment by otbe** [comment]
This is also an issue if you install `kscript` in a CI environment. We have images based on alpine and debian and both seems to fail with the new version.

**Comment by aartiPl** [comment]
This problem is already fixed in master. Unfortunately, I can not do the releases yet. @holgerbrandl - can you please help here?

**Comment by aartiPl** [comment]
Should be fixed in kscript 4.1.0. Please recheck and reopen the ticket if needed.
